schemas/latest/dbt_yml_files-latest.json

```diff
@@ -1214,17 +1162,39 @@
           "type": "string"
         },
         "expr": {
-          "type": [
-            "string",
-            "boolean"
-          ]
+          "type": ["string", "boolean"]
         },
         "role": {
           "type": "string"
         }
       },
       "additionalProperties": false
     },
+    "export": {
+      "type": "object",
+      "required": ["name"],
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "config": {
+          "type": "object",
+          "properties": {
+            "alias": {
+              "type": "string"
+            },
+            "export_as": {
+              "enum": ["table", "view", "cache"]
+            },
+            "schema": {
+              "type": "string"
+            }
+          },
+          "additionalProperties": true
```

Comment:
I think this breaks autocomplete or something? So i lean towards being false and we keep up to date. Without evidence, it's a weak opinion weakly held.